### PR TITLE
Change monitor log

### PR DESCRIPTION
**Pull request by crisingulani**

### CODE REVIEW

**Comment by crisingulani** [comment]
Ops... Yes, I did not add a new key (logpipeline) in the template.

File: framework/qlf/ui_channel/upstream.py [hunk 1]
```diff
@@ -112,13 +141,13 @@ def get_current_state():
     daemon_status = qlf.get_status()
 
     logfile = open_file('logfile')
-    ingestionlog = []
+    pipelinelog = str()
```
Comment (felipelm) [inline framework/qlf/ui_channel/upstream.py]: This breaks `qlf-ui` as it expects a `Array` not a `String`

File: framework/qlf/ui_channel/upstream.py [hunk 2]
```diff
@@ -58,26 +64,49 @@ def get_camera_log(cam):
             log = arq.readlines()
             return log
 
-    except Exception as e:
-        print(e)
+    except Exception as err:
+        logger.error(err)
         return "Error"
```
Comment (felipelm) [inline framework/qlf/ui_channel/upstream.py]: Here return `return ['get_pipeline_log Error']` so that it doesn't break `qlf-ui`.

**Comment by crisingulani** [comment]
How is your configuration file?

File: framework/qlf/ui_channel/upstream.py (hunk 2 above)
Comment (crisingulani) [reply framework/qlf/ui_channel/upstream.py]: How is your configuration file?

**Pull request merged**